cds_sorenson/api.py

```diff
@@ -27,31 +27,39 @@
 from __future__ import absolute_import, print_function
 
 import json
+from collections import OrderedDict
+from itertools import chain
 
 import requests
 from flask import current_app
 
-from .error import SorensonError
+from .error import InvalidAspectRatioError, InvalidResolutionError, \
+    SorensonError
 from .utils import generate_json_for_encoding, get_status
 
 
-def start_encoding(input_file, preset_ID, output_file=None):
+def start_encoding(input_file, output_file, preset_quality,
+                   display_aspect_ratio, **kwargs):
     """Encode a video that is already in the input folder.
 
     :param input_file: string with the filename, something like
         /eos/cds/test/sorenson/8f/m2/728-jsod98-8s9df2-89fg-lksdjf/data where
         the last part "data" is the filename and the last directory is the
         bucket id.
-    :param preset_ID: id of the preset.
     :param output_file: the file to output the transcoded file.
+    :param preset_quality: quality of the preset.
+    :param display_aspect_ratio: the video's aspect ratio
+    :param kwargs: other technical metadata
```

Comment:
It is the API form that we'll use later, when we use this API for `invenio-videos`. The nice thing is that the user/overlay will just drop all metadata information on `kwargs` without knowledge of the current backend.